dataprofiler/profilers/float_column_profile.py

```diff
@@ -50,6 +49,14 @@ def __add__(self, other):
         merged_profile = FloatColumn(None)
         BaseColumnPrimitiveTypeProfiler._add_helper(merged_profile, self, other)
         NumericStatsMixin._add_helper(merged_profile, self, other)
+        
+        self._merge_calculations(merged_profile.__calculations,
+                                 self.__calculations,
+                                 other.__calculations)
+
+        if "precision" in merged_profile.__calculations:
+            merged_profile.precision = max(self.precision, other.precision)
```

Comment:
Do we want consistency with merging or should in this case we always assume that max is what we want for precision? Actually, Now that I think about it, max likely is not correct. As usually precision is set to the lowest precision visible. We should have to correct this  in `_update_precision`

Now since we are using this functionality in two places, should we actually have a _merge_precision function? and then both calls use it?

Reply:
Updated this to be min. Actually updated precision in general to be more appropriate.
I don't see a need to create a one line function tbh